embedded/imu_driver/imu_driver.cc

```diff
@@ -70,6 +70,15 @@ bool ImuDriver::initialize() {
   // TODO(jake): Figure out if we can get an "i2c ready" flag
   std::this_thread::sleep_for(std::chrono::milliseconds(500));
 
+
+  bno_->setMode(Adafruit_BNO055::adafruit_bno055_opmode_t::OPERATION_MODE_AMG);
+  Adafruit_BNO055::adafruit_bno055_opmode_t mode = bno_->getMode();
+
+  while(mode != Adafruit_BNO055::adafruit_bno055_opmode_t::OPERATION_MODE_AMG) {
+    bno_->setMode(Adafruit_BNO055::adafruit_bno055_opmode_t::OPERATION_MODE_AMG);
+    mode = bno_->getMode();
+    std::this_thread::sleep_for(std::chrono::milliseconds(1));
+  }
```

Comment:
- How did you verify that this works? 
- Put this in a member function

Reply:
I moved the function into a member function.
I verified that it works by testing on the jet and verifying that we don't see 0s coming out anymore.  Not sure how else I can prove that it works?